build.rs

```diff
@@ -1,5 +1,7 @@
 fn main() {
-    system_deps::Config::new().probe().unwrap();
+    if !cfg!(windows) {
```

Comment:
There is already a similar check in Cargo.toml file to run system-deps conditionally only on non-Windows platforms, so one of these occurrences should be removed.

Reply:
https://github.com/tomm/fab-agon-emulator/blob/e1d5197a5f27268755a0cbad29de0db4ef9f7d19/Cargo.toml#L15